de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties

```diff
@@ -951,3 +951,31 @@ label.dateTimeWithPrecision.day_capitalized = Tag
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nIhr Passwort wurde geändert.\n\nDas mit dem Account ${userAccount.emailAddress} verknüpfte Passwort wurde geändert.\n\nVielen Dank.
+
+confirm_email_changed_email_html_text = <p>\n Sehr geehrte ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Sie haben kürzlich die E-Mail-Adresse geändert, die mit \n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Vielen Dank.\n </p>
+confirm_email_changed_email_plain_text = Sehr geehrte ${userAccount.firstName} ${userAccount.lastName}\n\nSie haben kürzlich die E-Mail-Adresse geändert, die mit\n${userAccount.firstName} ${userAccount.lastName}\nVielen Dank.
+
+first_time_external_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Herzlichen Glückwunsch!</strong>\n </p>\n\n <p>\n Wir haben einen Account für Sie angelegt. Ihr Account ist mit der E-Mail-Adresse ${userAccount.emailAddress} verknüpft.\n </p>\n\n <p>\n Vielen Dank!\n </p>\n
+first_time_external_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nHerzlichen Glückwunsch!\n\nWir haben einen Account für Sie angelegt. Ihr Account ist\nmit der E-Mail-Adresse ${userAccount.emailAddress} verknüpft.\n\nVielen Dank!
+
+password_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Passwort erfolgreich erstellt.</strong>\n </p>\n\n <p>\n Ihr Passwort für den Account ${userAccount.emailAddress} wurde erstellt.\n </p>\n\n <p>\n Vielen Dank.\n </p>
+password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPasswort erfolgreich erstellt.\n\nIhr Passwort für den Account ${userAccount.emailAddress}\nwurde erstellt.\n\nVielen Dank.
+
+application_error_email_html_text = <p>Zum Zeitpunk ${datetime!} ist ein Fehler auf der Webseite ${siteName!} aufgetreten.</p>\n\n <p>\n <strong>${i18n().requested_url}:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>${i18n().error_message}:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>${i18n().stack_trace}</strong> (${i18n().trace_available(siteName!)}):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>${i18n().caused_by}:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>
+
+application_error_email_plain_text = Zum Zeitpunk ${datetime!} ist ein Fehler auf der Webseite ${siteName!} aufgetreten.\n${i18n().requested_url}: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n ${i18n().error_message}: ${errorMessage!}\n</#if>\n\n${i18n().stack_trace} (${i18n().trace_available(siteName!)}):\n${stackTrace!}\n\n<#if cause?has_content>\n${i18n().caused_by}:\n${cause!}\n</#if>
```

Comment:
here are still i18n().requested_url, etc.

Reply:
Fixed that